Extensions/blacklist.js

```diff
@@ -744,21 +735,16 @@ XKit.extensions.blacklist = new Object({
 		}
 
 		$(obj).addClass("xblacklist_blacklisted_post");
-		$(obj).find(".post_info").css("display", "none");
-		$(obj).find(".post_controls").css("display", "none");
-		$(obj).find(".post_content").html(old_content + block_excuse);
-		$(obj).find(".post_footer_links").css('display', 'none');
-		$(obj).find(".post_source").css('display', 'none');
-		$(obj).find(".post-source-footer").css('display', 'none');
+		$(obj).prepend(block_excuse);
 
 		if (XKit.extensions.blacklist.preferences.mini_block.value !== true) {
 			$(obj).addClass("xblacklist_blacklisted_post_full_ui");
 		}
 
 		if (XKit.extensions.blacklist.preferences.show_tags.value === true && XKit.extensions.blacklist.preferences.mini_block.value === false) {
-			$(obj).find(".post_footer").css('display', 'none');
-		} else {
-			$(obj).find(".post_tags, .post_footer").css('display', 'none');
+			const tagsSel = XKit.css_map.keyToCss('tags') || '.post_tags';
+			const excuseTags = $(obj).find(tagsSel);
+			$(obj).find('.xblacklist_excuse').append(excuseTags);
```

Comment:
this *moves* the tags to the blacklist excuse, meaning tags disappear from the unhidden post. easily fixed by using `excuseTags.clone()`

Reply:
Oh woops I had clone in at one point then deleted it because I thought I was being paranoid